js/index.js

```diff
@@ -1,5 +1,79 @@
 // Iteration 1: Names and Input
-console.log("I'm ready!");
+let hacker1 = "Linux";
+console.log(`The driver's name is ${hacker1.toLowerCase()}`);
+let hacker2 = "DuckDuckGo";
+console.log(`The navigator's name is ${hacker2.toLowerCase()}`);
+
 // Iteration 2: Conditionals
+if(hacker1.length > hacker2.length){
+	console.log(`The ${hacker1} has the longest name, it has ${hacker1.length} characters.`);
+} else if(hacker1.length < hacker2.length) {
+	console.log(`It seems that the ${hacker2} has the longest name, it has ${hacker2.length} characters`);
+} else {
+	console.log(`Wow, you both have equally long names, ${hacker1.length} characters!`);
+}
 
 // Iteration 3: Loops
+let name = "Jhon";
```

Comment:
este te corre todas las declaraciones no aplica seria meter un for dentro de otro pero no creo que sea la manera igual veremos mas anidados despues en los juegos pero buena intencion